R/imagesFromVideos.R

```diff
@@ -18,21 +18,20 @@
 #' \dontrun{
 #' frames <- imagesFromVideos(videos, outdir = "C:\\Users\\usr\\Videos\\", frames = 5)
 #' }
-imagesFromVideos <- function(files, outdir = tempfile(), outfile = NULL, format = "jpg", fps = NULL, frames = NULL, parallel = FALSE, nproc = 1) {
-  if (!is(files, "data.frame")) {
-    stop("Error: 'mdresults' must be Data Frame.")
-  }
-  if (outdir != "" & !dir.exists(outdir)) {
-    if (!dir.create(outdir, recursive = TRUE)) {
-      stop("Output directory invalid.\n")
+imagesFromVideos <- function(files, outdir = tempfile(), outfile = NA, format = "jpg", fps = NULL, frames = NULL, parallel = FALSE, nproc = 1) {
+  if (!is.na(outfile) && file.exists(outfile)) {
```

Comment:
this exists in 2+ places, lets make it a function

Reply:
Would it be okay to add it to the loadData function? So it checks for the files existence and prompt the user if it exists, or make it a separate function within the same file?